### DISCUSSION TITLE
Instruction video for this project

### DISCUSSION THREAD

**Discussion by younoch** [hbollon/portfolio-vuejs#15] [Q&A]
Hey,
I'm a new vue developer. So It's harder to manage this project. Could you suggest any video tutorial for me?
Thanks advance.

Regards,
Younoch

**Comment by hbollon** [comment]
Hi! 
I'm not a heavy consumer of video tutorials, so I don't have one for you to recommend.
You're struggling to start this project or rather to maintain it because of Vue?

If it's to getting started with this project you just have to create a CosmicJS bucket (following the structure I used) and link it to the project.
To simplify this process I added to this repository [an export](https://github.com/hbollon/portfolio-vuejs/blob/master/data/portfolio-vuejs_import.json) of mine that you can import directly into CosmicJS (it will be pre-filled with my information, you will only have to edit it via the site interface) .
Digging around quickly, I found this article that explains how to do a little better than what's in the Readme with bonus animation: https://www.cosmicjs.com/articles/how-to-import-and-export-your-cosmic-js-bucket-jdlzwxer
Once done, you will only have to modify the `.env` file as explained in the [Readme](https://github.com/hbollon/portfolio-vuejs/blob/master/README.md)! :smile: 

Otherwise, if your problem is with Vue, I strongly recommend the [official documentation](https://vuejs.org/v2/guide/) (this project was done with version 2.0 of the framework, be careful not to base yourself on the one for Vue 3)
